app/views/compute_resources_vms/form/_openstack.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 <%= select_f f, :tenant_id, compute_resource.tenants, :id, :name, {}, :label => _('Tenant') %>
 
-<%= select_f f, :security_groups, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
+<%= select_f f, :security_group, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
```

Comment:
Unfortunately I am not a developer so I wont be able to help much, but I can do testing. At least with Icehouse and foreman 1.7.x right now.